app/models/vm.rb

```diff
@@ -1,6 +1,7 @@
 class Vm < VmOrTemplate
   default_scope { where(:template => false) }
-
+  has_one :container_deployment, :through => :deployments_node_deployment
```

Comment:
Should be `:through => :container_deployment_node`
And shouldnt it be `has_many :container_deployments` ?